howso/utilities/feature_attributes/base.py

```diff
@@ -950,6 +957,77 @@ def _is_iso8601_datetime_column(self, feature: str) -> bool:
         # No issues; it's valid
         return True
 
+    def _is_json_feature(self, feature: str) -> bool:
+        """
+        Return whether the given feature contains valid JSON.
+
+        Parameters
+        ----------
+        feature: string
+            The feature to check the values of.
+
+        Returns
+        -------
+        True if the column values can be parsed into JSON.
+        """
+        first_non_none = self._get_first_non_null(feature)
+        if first_non_none is None:
+            return False
+
+        # Sample 100 random values
+        for _ in range(30):
+            rand_val = self._get_random_value(feature, no_nulls=True)
+            if rand_val is None:
+                return False
+
+            # Try to parse rand_val as JSON
+            try:
+                if all([
```

Comment:
Only concern about this is if users are suddenly interested in using JSON lists as feature values, but in that case I guess they can always manually specify the feature type. Just want to raise this concern in case others agree.